app/views/questions/notification_preference/edit.html.erb

```diff
@@ -24,7 +24,7 @@
         </div>
         <div class="text-input--small-label question-with-follow-up__follow-up" id="sms-opt-in">
           <div class="form-group">
-            <%= f.cfa_input_field(:sms_phone_number, t("views.questions.notification_preference.options.sms_phone_number"), type: "tel", classes: ["form-width--long"]) %>
+            <%= f.cfa_input_field(:sms_phone_number, t("general.sms_phone_number"), type: "tel", classes: ["form-width--long"]) %>
```

Comment:
This is unrelated to the purpose of the pull request, but we talked to Anu and it seemed like a good change.